File: src/main/java/uk/gov/justice/hmpps/offenderevents/OffenderEventsApplication.java
package uk.gov.justice.hmpps.offenderevents;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;

@SpringBootApplication
@EnableResourceServer
public class OffenderEventsApplication {
    public static void main(final String[] args) {
        SpringApplication.run(OffenderEventsApplication.class, args);
    }

}

File: src/main/kotlin/uk/gov/justice/hmpps/offenderevents/OffenderEventsApplication.kt
package uk.gov.justice.hmpps.offenderevents

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class OffenderEventsApplication

fun main(args: Array<String>) {
  runApplication<OffenderEventsApplication>(*args)
}
